src/mlpack/tests/load_save_test.cpp

```diff
@@ -2651,7 +2651,6 @@ TEST_CASE("LoadCSVSemiColonMissingToNanHeader", "[LoadSaveTest]")
 
   data::Load("test.csv", dataset, opts);
 
-  dataset.print();
```

Comment:
Sorry for this

Reply:
No worries!  I have done the same many times over the years and only found it later :smile: